EST.MIT.InvoiceImporter.Function.Test/EST.MIT.InvoiceImporter.Function.Test.csproj

```diff
@@ -36,9 +36,8 @@
     <ProjectReference Include="..\EST.MIT.InvoiceImporter.Function\EST.MIT.InvoiceImporter.Function.csproj" />
   </ItemGroup>
 
-
   <ItemGroup>
-    <None Include="TestData\MIT Invoice Import Spreadsheet.xlsx">
+    <None Update="TestData\MIT Invoice Import Spreadsheet.xlsx">
```

Comment:
This spreadsheet shouldn't be in source control

Reply:
Resolved